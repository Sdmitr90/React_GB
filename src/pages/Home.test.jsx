import React from 'react'
import { render, screen } from '@testing-library/react'
import userEvent from '@testing-library/user-event'
import '@testing-library/jest-dom'

import Home from './Home'
import Buttom from "../components/Button/Button";

describe('Home', () => {
    it('render component Home', () => {
        render(<Home>Home</Home>)
    })
    it('render with snapshot', () => {
        const {asFragment} = render(<Home>Home</Home>)
        expect(asFragment()).toMatchSnapshot()
    })
});