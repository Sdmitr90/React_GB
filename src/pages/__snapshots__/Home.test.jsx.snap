// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render with snapshot 1`] = `
<DocumentFragment>
  <div>
    Home
  </div>
</DocumentFragment>
`;
